docs/operating/troubleshooting.md

```diff
@@ -5,4 +5,35 @@ menu: operating
 slug: /troubleshooting.md
 ---
 
-Work in progress (:
+# Troubleshooting; Common cases
+
+
+## Overlaps
+
+Overlaps should never happen in healthy and well configured Thanos system. That's why there is no automatic repair.
+
+Let's take two examples:
+
+- `msg="critical error detected; halting" err="compaction failed: compaction: pre compaction overlap check: overlaps found while gathering blocks. [mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s, blocks: 2]: <ulid: 01D94ZRM050JQK6NDYNVBNR6WQ, mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s>, <ulid: 01D8AQXTF2X914S419TYTD4P5B, mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s>`
+
+It shows that you have really the same blocks for 2h overlaps. It worth checking whether are those blocks the same (check the number of series and samples, checksum the index and chunks).
```

Comment:
```suggestion
In this halted example, we can read that compactor detected 2 overlapped blocks. What's interesting is that those two blocks look like they are "similar". They are exactly for the same period of time. This might mean that potential reasons are:
* Duplicated upload with different ULID (non-persistent storage for Prometheus can cause this)
* 2 Prometheus-es are misconfigured and they are uploading the data with exactly the same external labels. This is wrong, they should be unique.

Checking producers log for such ULID, and checking meta.json (e.g if sample stats are the same or not) helps.  Checksum the index and chunks files as well to reveal if data is exactly the same, thus ok to be removed manually.
```